### DISCUSSION TITLE
[4RFC] com_menu add duplicaste - actions dropdown

### DISCUSSION THREAD

**Discussion by adj9** [joomla/joomla-cms#47146] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
aIn the wake of article duplication [(#28881),](https://github.com/joomla/joomla-cms/pull/28881) it may be convenient to question menu duplication with a different behavior.  This is to speed up the development of the site that have similar menus.

### Describe the solution you'd like
In the duplication of the menu, you can create a new one 
- Of the same type 
- With basic functionality, if, for example, 
-- the menu is for registered users, the duplicate is for public users. 
-- If it is a submenu, the duplicate will be a menu... 
This is to start with a basic menu
- With a different name, an incremental number is fine, like [(#28881),](https://github.com/joomla/joomla-cms/pull/28881)

**Comment by brianteeman** [comment]
For reference the article duplication PR referred to was closed without being merged.

As I wrote in that issue I dont see the need for this as you can already achieve the same by using "Save as Copy"
